components/menu-dropdown/menu-dropdown.jsx

```diff
@@ -662,7 +682,7 @@ class MenuDropdown extends React.Component {
 		const isOpen = this.getIsOpen();
 
 		if (!isOpen) {
```

Comment:
Since this isn't needed anymore, let's remove it.

Reply:
We might need this to support the `onFocus` prop for now.

Reply:
Right.  I meant remove lines 662-666 (left side).